packages/web/localizations/en.json

```diff
@@ -15,6 +15,7 @@
     "available": "Available",
     "MAX": "MAX",
     "HALF": "HALF",
+    "DEPOSIT": "DEPOSIT",
```

Comment:
Please add `fr` translations as well. Deposit should be found elsewhere in the `fr.json` file.

Reply:
done